MaxText/configs/models/deepseek2-236b.yml

```diff
@@ -13,7 +13,6 @@
 # limitations under the License.
 
 # model config for DeepSeek V2 - 236B
-# Please note: DeepSeek-style is not fully support at this moment
```

Comment:
We haven't yet tested this config, should we keep this like?

Reply:
Oh, that's a good point! Will add it back.